src/react-components/home-root.js

```diff
@@ -186,22 +188,51 @@ class HomeRoot extends Component {
     };
   };
 
+  launchTour = () => {
+    createAndRedirectToNewHub(
+      "Hubs Tour",
+      // TODO BP placeholder tour scene
+      // "chjVRLh", // dev
+      "d2SF68V", // prod
+      null,
+      false
+    );
+  };
+
+  showCreateDialog = () => {
+    this.showDialog(
+      props => (
+        <DialogContainer title="Choose a Scene" {...props}>
+          <HubCreatePanel {...props} />
+        </DialogContainer>
+      ),
+      {
+        initialEnvironment: this.props.initialEnvironment,
+        environments: this.state.environments
+      }
+    );
+  };
+
   render() {
     const mainContentClassNames = classNames({
       [styles.mainContent]: true,
       [styles.noninteractive]: !!this.state.dialog
     });
 
+    const isOculusBrowser = /Oculus/.test(navigator.userAgent) && qsTruthy("enable_ftue");
```

Comment:
would probably rename to `useOculusBrowserFTUE` or something, i would have assumed this was a browser check